Files/Views/ModernShellPage.xaml.cs

```diff
@@ -207,6 +212,42 @@ public ModernShellPage()
             AppServiceConnectionHelper.ConnectionChanged += AppServiceConnectionHelper_ConnectionChanged;
         }
 
+        private void InitializeCommands()
+        {
+            if (this.SlimContentPage != null)
+            {
+                SelectAllContentPageItemsCommand = new RelayCommand(this.SlimContentPage.SelectAllItems);
+                ClearContentPageSelectionCommand = new RelayCommand(this.SlimContentPage.ClearSelection);
+                InvertContentPageSelctionCommand = new RelayCommand(this.SlimContentPage.InvertSelection);
+            }
+            PasteItemsFromClipboardCommand = new RelayCommand(async () => await UIFilesystemHelpers.PasteItemAsync(FilesystemViewModel.WorkingDirectory, this));
+            CopyPathOfWorkingDirectoryCommand = new RelayCommand(CopyWorkingLocation);
+            OpenNewWindowCommand = new RelayCommand(NavigationHelpers.LaunchNewWindow);
+            OpenNewPaneCommand = new RelayCommand(() => PaneHolder?.OpenPathInNewPane("NewTab".GetLocalized()));
+            OpenDirectoryInDefaultTerminalCommand = new RelayCommand(() => NavigationHelpers.OpenDirectoryInTerminal(this.FilesystemViewModel.WorkingDirectory, this));
+            AddNewTabToMultitaskingControlCommand = new RelayCommand(async () => await MainPage.AddNewTabByPathAsync(typeof(PaneHolderPage), "NewTab".GetLocalized()));
```

Comment:
Do we need a reference to every command? Can we just have a reference to the view model?

Reply:
Currently we don't have any view model in ModernShellPage (?)
Changing this would go well over the scope of this PR @yaichenbaum